x/fees/types/fee_test.go

```diff
@@ -0,0 +1,146 @@
+package types
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/tharsis/ethermint/tests"
+)
+
+type FeeTestSuite struct {
+	suite.Suite
+}
```

Comment:
you can probably add the `addressN` fields here and remove them from the tests funcs below 👍